src/index.html

```diff
@@ -5,9 +5,44 @@
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <title>Product cards</title>
-    <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="../src/styles/index.css">
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div class="product">
+      <div class="product-card" data-qa="card">
+        <div class="product-image-container">
+          <img class="product-image" src="../src/images/imac.jpeg"
+          alt="image">
```

Comment:
Need to check and fix it everywhere
<img width="1057" alt="image" src="https://github.com/mate-academy/layout_product-cards/assets/64750958/b174d3af-99e4-4796-8685-4fcdebb08db2">
```suggestion
          <img
            class="product-image" 
            src="../src/images/imac.jpeg"
            alt="image"
          >
```